.github/CHANGELOG.md

```diff
@@ -1,22 +1,31 @@
-# Release 0.13.0-dev (development release)
+# Release 0.13.0rc0 (development release)
 
 <h3>New features since last release</h3>
 
 * Adds initial support for the Xanadu's photonic quantum hardware.
   [(#101)](https://github.com/XanaduAI/strawberryfields/pull/101)
   [(#148)](https://github.com/XanaduAI/strawberryfields/pull/148)
   [(#294)](https://github.com/XanaduAI/strawberryfields/pull/294)
+  [(#327)](https://github.com/XanaduAI/strawberryfields/pull/327)
+  [(#328)](https://github.com/XanaduAI/strawberryfields/pull/328)
+  [(#329)](https://github.com/XanaduAI/strawberryfields/pull/329)
+  [(#330)](https://github.com/XanaduAI/strawberryfields/pull/330)
+  [(#334)](https://github.com/XanaduAI/strawberryfields/pull/334)
+  [(#336)](https://github.com/XanaduAI/strawberryfields/pull/336)
+  [(#337)](https://github.com/XanaduAI/strawberryfields/pull/337)
+  [(#339)](https://github.com/XanaduAI/strawberryfields/pull/339)
 
   Jobs can now be submitted to the Xanadu cloud platform to be run
   on supported hardware using the new `RemoteEngine`:
 
   ```python
-  from strawberryfields import RemoteEngine
+  import strawberryfields as sf
   from strawberryfields import ops
   from strawberryfields.utils import random_interferometer
 
   # replace AUTHENTICATION_TOKEN with your Xanadu cloud access token
-  eng = RemoteEngine("chip2", token="AUTHENTICATION_TOKEN")
+  con = sf.api.Connection(token="AUTH_TOKEN")
```

Comment:
(Edit: removed my previous comment here as I was using the wrong SF :sweat_smile: )